src/upload.ts

```diff
@@ -50,10 +54,20 @@ async function uploadToDir (filepath: string, dir: string) {
   return destination
 }
 
+async function uploadToSQLite (filepath: string, databaseFile: string) {
+  const { initStorage, writeData } = await import('./database-sqlite.ts')
+  initStorage(databaseFile)
+  const buffer = await Deno.readFile(filepath)
+  const hash = blake32Hash(buffer)
+  writeData(`blob=${hash}`, buffer)
```

Comment:
It seems like this could result in a conflict between the two PRs?

If we're always storing files as blobs here, even if some of them aren't, then wouldn't that conflict with the way [`readData` is implemented here](https://github.com/okTurtles/group-income/pull/1527#pullrequestreview-1388983681)?

Reply:
If we're always storing files as blobs here then `readData() `will have to be called used with `blob=` keys to get the data back. So the callers would need to be updated, rather than  `readData() ` itself I think.
We can also store files as text, but that will limit `chel deploy` to text files

Reply:
Ok, thinking on this more, I think there is no conflict.

Reply:
@snowteamer Looking at it closer, https://github.com/okTurtles/group-income/pull/1552 does seem to not store with the prefix, so can we remove the prefix here?